app/main.py

```diff
@@ -2,24 +2,66 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: type, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: type, owner: type) -> int:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: type, value: int) -> None:
+        if self.min_amount <= value <= self.max_amount:
+            setattr(instance, self.protected_name, value)
+        else:
+            raise ValueError
```

Comment:
It's a good practice to provide a message when raising an exception. This will help you understand what went wrong if an exception is raised.